solidity/random-beacon/contracts/RandomBeaconChaosnet.sol

```diff
@@ -15,28 +15,19 @@
 pragma solidity 0.8.17;
 
 import "./api/IRandomBeacon.sol";
-import "./libraries/Callback.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
 /// @title Keep Random Beacon Chaosnet Stub
 /// @notice Keep Random Beacon stub contract that will be used temporarily until
 ///         the real-world random beacon client implementation is ready.
-/// @dev Used for testing purposes only.
 contract RandomBeaconChaosnet is IRandomBeacon, Ownable {
-    using Callback for Callback.Data;
-
-    /// @notice Relay entry callback gas limit.
-    uint256 public constant _callbackGasLimit = 64_000;
-
     /// @notice Authorized addresses that can request a relay entry.
     mapping(address => bool) public authorizedRequesters;
```

Comment:
Instead of mapping, I think we should pass `address authorizedRequester` in the constructor. This will be the address of `Bridge` contract proxy. We can make this field `immutable` which will lower the gas cost. We will then require `msg.sender` to be the requester.

Reply:
Unfortunately, it seems `RandomBeaconChaosnet` is deployed before `WalletRegistry`. Therefore we don't know the address of `WalletRegistry` when we are deploying `RandomBeaconChaosnet`.

Reply:
@pdyraga Should I replace the mapping with a single address, though?
This can be done.

Reply:
You are right. Let's leave it as-is, it's a minor thing.